rules/examples/Invoice reponse use cases/T111-uc007-Payment has been initiated.xml

```diff
@@ -40,4 +40,8 @@
 			<cbc:DocumentTypeCode listID="UNCL1001">380</cbc:DocumentTypeCode>
 		</cac:DocumentReference>
 	</cac:DocumentResponse>
+	<cac:DocumentReference>
```

Comment:
This Addition to the example does not validate against the IMR Schematron. Consider moving the DocumentReference Inside the DocumentResponse or fix the syntax rules of IMR

Reply:
I was trying to improve the clarity of the documentation by showing the two business terms Invoice id and Payment id, separately instead of combining them into the same element. That does not seem to work because Travis builds the syntax validation rules from this and when the same element is shown twice it does not seem to work out. The issue is that the syntax is 1..1, default but when adding the option to give payment reference the cardinality needs to be extened. Since the intention is to support two identifiers and not unbounded and one of those identifiers, the reference to the invoice, as mandatory and the payment reference is optional up to once the suitable cardinality. To solve this I removed the dublicate document referencd syntax and change the cardinality of the existing one to 1..2. The description and business term identifier are then merged. This however means that additional rules are needed to make sure that the invoice reference is included at least once otherwise it would be possible to create a message with payment reference only. Suggest however to merge this change as is, if it works, and add the rules in a new branch.